packages-ts/gauntlet-terra/src/commands/internal/terra.ts

```diff
@@ -139,13 +144,15 @@ export default abstract class TerraCommand extends WriteCommand<TransactionRespo
     return this.wrapResponse(res)
   }
 
-  async simulate(address, input): Promise<Number> {
-    const msg = new MsgExecuteContract(this.wallet.key.accAddress, address, input)
+  async simulate(signer: AccAddress, contractAddress: AccAddress, input: any): Promise<Number> {
```

Comment:
I think would make sense this to accept a list of messages instead of a contract and an input, something like the [signAndSend](https://github.com/smartcontractkit/chainlink-terra/blob/develop/packages-ts/gauntlet-terra/src/commands/internal/terra.ts#L68) function.
This would the function also easier to use from the multisig (or any other consumer)
```suggestion
  async simulate(signer: AccAddress, msgs: MsgExecuteContract[]): Promise<Number> {
  ...
  const tx = await this.wallet.createTx({
      msgs,
  })
```